doc/config_example.txt

```diff
@@ -0,0 +1,32 @@
+tempX=1e-3                          # plasma electron temperature along X
+tempY=1e-3                          # plasma electron temperature along Y
+tempZ=1e-3                          # plasma electron temperature along Z
+beamImp=0.2                         # beam impulse
+beamVelDisp=0.14                    # beam velocity dispersion
+beamPlasmaDensityRat=2.0e-3         # beam and plasma density ratio
+plsmDensity=1.0                     # plasma density
+externalMagnFieldX=8.0              # external magnetic field (along X)
+lx=1.1424                           # domain size X
+ly=0.05                             # domain size Y
+lz=0.05                             # domain size Z
+px=12                               # plasma size X
+py=13                               # plasma size Y
+bx=14                               # beam size X
+by=15                               # beam size Y
+bz=16                               # beam size Z
+lp=1000                             # average number of particles in cell
+nx=100                              # number of mesh nodes along X
+ny=4                                # number of mesh nodes along Y
+nz=4                                # number of mesh nodes along Z
+tau=0.001                           # timestep
+beamPlasma=22                       # 1 if beam-plasma interaction, 0 if beam-beam
+startFromFile=23                    # moment to start from saved
+phase=24                            # phase to start from save
+ts=5                                # total steps
```

Comment:
Here you treat it as the last step, not a total steps
https://github.com/snytav/atom/pull/3/files/6f6ce44d997ae99ab3dc0f2d24e0a0c656d35b95#diff-bd43e2a2217c1d20d985891619aacdb3R1138

Reply:
changed to last step - _lst_